app/_src/gateway/plugin-development/pluginserver/python.md

```diff
@@ -77,6 +80,25 @@ class Plugin(object):
         host, err = kong.request.get_header("host")
 ```
 
+{:.important}
+> **Warning:** classes and functions in the `kong_pdk.pdk.kong` module cannot be used directly because they're only used for type hints. To call PDK functions, you need to use the `kong` object which is passed as the phase handler's parameter. Please keep in mind that if you want to call PDK functions outside of the phase handler, you'll also need to pass the `kong` object to your outer code.
```

Comment:
```suggestion
{:.important}
> **Warning:** Classes and functions in the `kong_pdk.pdk.kong` module cannot be used directly because they're only used for type hints. To call PDK functions,  use the `kong` object that is passed as the phase handler's parameter. Keep in mind that if you want to call PDK functions outside of the phase handler, you also need to pass the `kong` object to your outer code.
```